source/_integrations/weather.template.markdown

```diff
@@ -30,6 +30,8 @@ weather:
     condition_template: "sunny"
     temperature_template: "{{ states('sensor.temperature') | float}}"
     humidity_template: "{{ states('sensor.humidity')| float }}"
+    wind_bearing_template: "{{ state_attr('weather.home', 'wind_bearing') }}"
+    wind_speed_template: "{{ state_attr('weather.home', 'wind_speed') | float }}"
```

Comment:
State attributes are already natively typed, any reason for the added typecast here?

Reply:
No particular reason to be honest, mostly because it's used above with temperature and humidity. Do you think it would be less confusing without a cast?

Reply:
temperature and humidity are using the entity states, which are strings.

Reply:
Okay, will remove the cast. However, wind speed is not actually typed - I just checked my template weather and it does allow non-numeric values :)
![Screenshot 2021-03-05 at 15 08 59](https://user-images.githubusercontent.com/6134677/110134009-dd03ae00-7dc4-11eb-8357-2b1189fb43f9.png)

Reply:
That would be actually incorrect handling of values by the integration providing that information.